src/daemon/collectd.c

```diff
@@ -487,11 +484,6 @@ int notify_systemd (void)
         su_size = sizeof (sa_family_t) + strlen (notifysocket);
         if (su_size > sizeof (su))
             su_size = sizeof (su);
-#else
-	ERROR ("Systemd socket uses Linux abstract namespace notation (\"%s\"), "
```

Comment:
What's the reason for removing this fallback? ("systemd is runs only on Linux anyway" is not a valid answer ;)